### PR TITLE
CSS added to header, background and input

### PR DESCRIPTION
**Pull request by AaronD87**

### CODE REVIEW

**Comment by mehtaphysical** [comment]
add dist files to .gitignore

File: src/components/App.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  text-transform: uppercase;
+  font-size: 100px;
+  font-weight: 800;
+  letter-spacing: -3px;
+  line-height: 1;
+  text-shadow: #EDEDED 3px 2px 0;
+  position: relative;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
```
Comment (mehtaphysical) [inline src/components/App.css]: end files with a newline

File: src/components/SaveAs.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  margin-left: 10px;
+  cursor:pointer;
+  border-radius: 5px;
+}
+
+span {
+  margin-right: 10px;
+  font-family: 'Open Sans', sans-serif;
+  color: #555;
+}
```
Comment (mehtaphysical) [inline src/components/SaveAs.css]: end files with a new line

File: src/components/SaveAs.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+form {
```
Comment (mehtaphysical) [inline src/components/SaveAs.css:1]: use em or rem instead of px

File: src/components/App.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.header {
```
Comment (mehtaphysical) [inline src/components/App.css:1]: use em or rem instead of px

File: src/components/App.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  color: #555;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 100px;
```
Comment (mehtaphysical) [inline src/components/App.css]: hmmmm

**Review by mehtaphysical — APPROVED** [review]

**Review by MariahAdams — APPROVED** [review]

**Review by mehtaphysical — APPROVED** [review]

**Review by caripizza — APPROVED** [review]
Great job Aaron!

**Pull request merged**